lib/usd/hdMaya/CMakeLists.txt

```diff
@@ -0,0 +1,192 @@
+#==============================================================================
+# Configuration hdMaya 
+#==============================================================================
+
+set(TARGET_NAME hdMaya)
+
+option(HDMAYA_BUILD_TESTS "Build the maya-to-hydra tests" OFF)
+
+set(MTOH_CXX_STD "11"
```

Comment:
Well, we were testing compiling with c++14, since that's vfxplatform CY2018+ standard.  It's not really necessary, though - and if we want to keep it, it should be done everywhere.

Worth noting - Spiff announced that USD itself is planning to move to c++14 soon...

Reply:
Went ahead and just removed it for now - we can bring something back later at top level if desired:

https://github.com/Autodesk/maya-usd/pull/231/commits/7955af55b1349ac829d40ec05734b7ca74d766a4